assets/src/infrastructure/ui/layout/menu/MenuToggle.tsx

```diff
@@ -6,6 +6,7 @@ import { MenuToggleProps } from './types';
 const MenuToggle = React.forwardRef<typeof ChakraMenuButton, MenuToggleProps>(
 	({ children, variant = 'unstyled', ...props }, ref) => {
 		return (
+			// @ts-ignore
 			<ChakraMenuButton {...props} variant={variant} ref={ref}>
```

Comment:
Better to add it only for that prop in a separate line
```suggestion
			<ChakraMenuButton
				{...props}
				// @ts-ignore
				variant={variant}
				ref={ref}
			>
```

Reply:
good catch, updated :+1:

Reply:
thnx guys